CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ function(importLibraries)
     "Linux,Darwin,Windows:gflags"
     "Linux,Darwin,Windows:glog"
     "Linux,Darwin,Windows:googletest"
+    "Linux,Darwin,Windows:zlib"
```

Comment:
This is a change to note. libarchive now needs to be compiled with zlib in order to extract the jar files. See the `libraries/cmake/source/libarchive/CMakeLists.txt` file for details. This means that zlib now needs to be compiled prior to libarchive.